spec/unit/who_dis_syntax_error_spec.rb

```diff
@@ -18,25 +18,45 @@ module DeadEnd
       ).to eq(:end)
     end
 
-    it "determines the type of syntax error to be an unmatched pipe" do
-      source = <<~EOM
-        class Blerg
-          Foo.call do |a
-          end # one
+    context "determines the type of syntax error to be an unmatched pipe" do
```

Comment:
@schneems I noticed you are not really using `context` in the specs, would you prefer to keep these separate?

Reply:
My experience with contexts is that they become large and unwieldy and lose their usefulness and then when tests get moved around the indentation makes the diff larger. I'm not strictly against it though.